README.md

```diff
@@ -2,6 +2,8 @@
 are not yet stable.* The github issue tracker is the primary way to
 communicate with the developers.
 
+[![Build Status](https://github.com/flux-framework/PerfFlowAspect/actions/workflows/github-actions.yaml/badge.svg)](https://github.com/flux-framework/PerfFlowAspect/actions)
```

Comment:
I think this can be its own commit. (following flux code-development convention -- having each topical transaction as its own commit).